CRM/Upgrade/Incremental/php/FiveFive.php

```diff
@@ -46,40 +46,6 @@ public function setPreUpgradeMessage(&$preUpgradeMessage, $rev, $currentVer = NU
     // }
   }
 
-  /**
-   * Upgrade function.
-   *
-   * @param string $rev
-   */
-  public function upgrade_5_5_alpha1($rev) {
-    $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
-    $this->addTask('Entity File schema updates', 'entityFileSchemaUpdate');
-  }
-
-  /**
-   * dev/core#321 - Prevent duplicate entries in civicrm_entity_file
-   *
-   * @param \CRM_Queue_TaskContext $ctx
-   *
-   * @return bool
-   */
-  public static function entityFileSchemaUpdate(CRM_Queue_TaskContext $ctx) {
-    if (CRM_Core_BAO_SchemaHandler::checkIfIndexExists('civicrm_entity_file', 'index_entity_file_id')) {
-      // Delete any stray duplicate rows and add unique index to prevent new dupes and enable INSERT/UPDATE combo query
-      CRM_Core_DAO::executeQuery('
```

Comment:
@monishdeb so I don't want to add the indices but quite possibly this cleanup should still run?  I assume the data here is still invalid?

Also, we should squash before merging